src/Paket.Core/PaketConfigFiles/DependencyCache.fs

```diff
@@ -98,7 +98,13 @@ type DependencyCache (lockFile:LockFile) =
             installModel
             |> InstallModel.getLegacyReferences (TargetProfile.SinglePlatform framework)
             |> Seq.map (fun l -> l.Path)
-            |> Seq.map (fun path -> AssemblyDefinition.ReadAssembly path, FileInfo(path))
+            |> Seq.map (fun path -> 
+                try 
+                    (AssemblyDefinition.ReadAssembly path, FileInfo(path)) |> Some
+                with
+                | _ -> None
```

Comment:
I don't like that pattern, can we at least catch only the relevant exception or otherwise log the error as warning?